engine/src/flutter/shell/platform/embedder/embedder.h

```diff
@@ -1594,6 +1628,9 @@ typedef struct {
   // Array of string attributes associated with the `decreased_value`.
   // Has length `decreased_value_attribute_count`.
   const FlutterStringAttribute** decreased_value_attributes;
+  // The set of semantics flags associated with this node. Prefer to use this
+  // over `flags`.
+  FlutterSemanticsFlags* flags2;
```

Comment:
Could we update flag's comment with something that indicates embedders should move away from it?

Reply:
If the goal is to move off `flags` and replace it with `flags2`, then:
1. Definitely add a comment telling the reader to migrate to `flags2`
2. Consider renaming `flags` to `flags__unused__` to force anyone building against the embedder to migrate.

Even with the field rename, We _must_ continue populating the existing `flags` field from the new struct so that we don't break existing embedders who drop in a new engine shared lib.

Reply:
@hannah-hyj FYI, this still needs to be done :)